docs/_quickstart.asciidoc

```diff
@@ -0,0 +1,100 @@
+[float]
+=== Step 2: Configure the APM integration
+
+{ecloud} runs a hosted version of {integrations-server} that includes the APM Server.
+
+include::{obs-repo-dir}/observability/tab-widgets/add-apm-integration/content.asciidoc[tag=ess]
+
+[float]
+=== Step 3: Install APM agents
+
+APM agents are written in the same language as your service.
+To monitor a new service, you must install the agent and configure it with a service name,
+APM Server host, and Secret token.
+
+* **Service name**: The APM integration maps an instrumented service's name–defined in each {apm-agent}'s configuration–
+to the index that its data is stored in {es}.
+Service names are case-insensitive and must be unique.
+For example, you cannot have a service named `Foo` and another named `foo`.
+Special characters will be removed from service names and replaced with underscores (`_`).
+
+* **APM Server URL**: The host and port that APM Server listens for events on.
+This should match the host and port defined when setting up the APM integration.
+
+* **Secret token**: Authentication method for {apm-agent} and APM Server communication.
+This should match the secret token defined when setting up the APM integration.
+
+TIP: You can edit your APM integration settings if you need to change the APM Server URL
+or secret token to match your APM agents.
+
+include::{docdir}/legacy/tab-widgets/install-agents-widget.asciidoc[]
+
+[discrete]
+=== Step 4: View your data
+
+Back in {kib}, under {observability}, select APM.
+You should see application performance monitoring data flowing into the {stack}!
+
+NOTE: The built-in `apm_user` role is not compatible with the APM integration
+as it only provides read access to `apm-*` indices.
+For a list of indices users need access to, refer to
+{apm-guide-ref}/apm-data-streams.html[APM data streams]
+
+// [role="screenshot"]
+// include::{obs-repo-dir}/observability/images/kibana-apm-sample-data.png[{apm-app} with data]
+
+Not seeing any data? Review our list of {apm-guide-ref}/common-problems.html[common problems] for helpful tips.
+
+[discrete]
+=== What's next?
+
+* Now that data is streaming into the {stack}, take your investigation to a
+deeper level! Use https://www.elastic.co/observability[Elastic {observability}]
+to unify your logs, metrics, uptime, and application performance data.
+
+* Want to protect your endpoints from security threats? Try
+https://www.elastic.co/security[{elastic-sec}]. Adding endpoint protection is
+just another integration that you add to the agent policy!
+
+* Are your eyes bleary from staring at a wall of screens?
+{observability-guide}/create-alerts.html[Create alerts] and find out about
+problems while sipping your favorite beverage poolside.
+
+* Want Elastic to do the heavy lifting? Use {ml} to
+{observability-guide}/inspect-log-anomalies.html[detect anomalies].
+
+* Got everything working like you want it? Roll out your agent policies to
+other hosts by deploying {agent}s across your infrastructure!
```

Comment:
Should we remove this? I'd like to have as little confusion with APM agents & Elastic Agent on this page as possible.